build.ps1

```diff
@@ -37,11 +40,11 @@ function CrossGen([string] $runtime, [bool] $isSelfContained, [string] $publishT
     DownloadNupkg "https://www.nuget.org/api/v2/package/Microsoft.Build.Utilities.Core/15.1.1012" @("lib\netstandard1.3\Microsoft.Build.Utilities.Core.dll")  @("$selfContained")
 
     # Publish self-contained app with all required dlls for crossgen
-    dotnet publish .\src\WebJobs.Script.WebHost\WebJobs.Script.WebHost.csproj -r $runtime -o "$selfContained" -v q /p:BuildNumber=$buildNumber    
+    & $dotnetx86 publish .\src\WebJobs.Script.WebHost\WebJobs.Script.WebHost.csproj -r $runtime -o "$selfContained" -v q /p:BuildNumber=$buildNumber    
```

Comment:
We shouldn't need to change all the publish/build steps, just the test runs, which are currently defined in the appveyor.yml file (would be a good idea to break that off). Right?